### PR TITLE
Chris 

### PR DESCRIPTION
**Pull request by ciredale**
Pulling to get the homework

### CODE REVIEW

**Comment by ciredale** [comment]
Submitting the completed assignment for Homework 4

**Comment by ciredale** [comment]
Trying to submit the homework again

**Comment by debbieyuen** [comment]
@ciredale this is okay, i can grade it as is! thank you chris!

File: hw04/BarbieWalletBalance.cs [hunk 1]
```diff
@@ -16,12 +16,48 @@ public class BarbieBank : MonoBehaviour
...
     {
-        int[] biggerWallet = BarbieWallet(500, 600);
+
+        //Question 2- Instantiate a new career for Barbie 
+        BarbieWorld<string> barbieWorld = new BarbieWorld<string>();
+        barbieWorld.BarbieCareers("GameDesigner");
+
+    //Problem 3
+        // Convert the existing function to a generic array 
+        var biggerWallet = BarbieWallet(500, 600);
```
Comment (debbieyuen) [inline hw04/BarbieWalletBalance.cs:26]: This problem is very close. Needs an open and close. When we call this function we want to pass in a function. We want to pass in some type - here it is an `int`.

```
    void Start()
    {
        int[] biggerWallet = BarbieWallet<int>(500, 600);
        Debug.Log(biggerWallet.Length + " " + biggerWallet[0] + " " + biggerWallet[1]);

        GetMoney<int>(500, 600);
    }

    // Generic function for BarbieWallet
    private T[] BarbieWallet<T>(T numOfPennies, T cashAmount)
    {
        return new T[] { numOfPennies, cashAmount };
    }

    // BarbieBank generic function
    private T[] BarbieBank<T>(T numOfPennies, T cashAmount, T numOfCreditCards)
    {
        return new T[] { numOfPennies, cashAmount, numOfCreditCards };
    }
```

**Review by debbieyuen — COMMENTED** [review]
thank you Chris!! :)

**Pull request closed without merging**